python/setup.py

```diff
@@ -298,6 +298,10 @@ def get_outputs(self):
     use_scm_version = {"root": "..", "relative_to": __file__},
     setup_requires=['setuptools_scm'],
     install_requires=['cython >= 0.23', 'numpy >= 1.9', 'six >= 1.0.0'],
+    test_requires=['pytest'],
+    extras_requires={
+        'parquet': ['parquet-cpp']
```

Comment:
This will be weird as there is not and probably won't be a parquet-cpp package on pypi. It is only available in conda.